### ISSUE TITLE
Electron Dev Tools "Expected ';'" Error in IE11

### ISSUE THREAD

**Issue by miking-the-viking** [vuejs/vue-devtools#626]
### Version
4.1.3

### Browser and OS info
IE 11/Windows 10

### Steps to reproduce
Reviewing my Vue app on IE11/Windows 10 and whenever the electron vue-devtools app is included it throws that error. Seems to be the syntax around the `var section of sections` here:

```
function get (object, path) {
  var sections = path.split('.')
  for (var section of sections) {
    object = object[section]
    if (!object) {
      return undefined
    }
  }
  return object
}

```

### What is expected?
vue-devtools script to run

### What is actually happening?
Script is crashing

---
Again it's IE1. I'm just documenting an issue I've noted (and wondering if anyone has already solved it). It's a dying breed that shouldn't be considered necessary.

**Comment by michalsnik** [comment]
Hey, thanks for posting this issue @miking-the-viking. It should work on IE11 and few other mobile browsers, so this seems like an unwanted side-effect of some changes in util loaded in `backend` script.

We can either replace this `for of` statement with supported loop, or add an extra build step before publishing package, so that the script loaded by your page is already properly transpiled.

For now though you might be ok with simply using https://babeljs.io/docs/plugins/transform-es2015-for-of/ in your application (if this is the case, and there are no other incompatibilities).

I can't inspect IE11 at this moment, so if you'd like to work on this once you confirm the origin of this problem - it'd be great :)

**Comment by Alendorff** [comment]
I faced with that error too and babel plugin doesn't help. I added vue-devtools as dev dependency and imported it in my main.js file, and used `connect()` function, also I installed babel plugin which you mentioned, but it doesn't help. I still see this annoying
![image](https://user-images.githubusercontent.com/4989157/43472021-b300f310-94ec-11e8-9f46-48e23b2979aa.png)
and page even doesn't load. 

Frustrating...
